app/controllers/api/v1/emotions_controller.rb

```diff
@@ -44,7 +44,11 @@ def additional_params
       response: @current_response ? response_hash : { attributes: { steps: %w[emotion-selection-web].to_s } },
       emotion: @current_response ? @current_response.emotion : {},
       api_giphy_key: ENV['GIPHY_API_KEY'].presence,
-      users: User.ordered.map { |user| { id: user.id, display: user.first_name } }
+      users: User.ordered.map do |user|
+        { id: user.id, display: user.first_name, first_name: user.first_name, last_name: user.last_name }
+      end,
+      my_shout_outs_to_other: Shoutout.where(user_id: current_user.id),
+      other_shout_outs_to_me: list_shoutouts_to(current_user)
```

Comment:
you don't need to send current_user in params of method because it available in any place of this class

Reply:
1. I have many options, but I think these names will be better for the rest of the team to understand. What I can to do I can to discuss it with our team. Although best decision probably are` user_shoutouts `and `shoutouts_to_user`
2.     I agree with you but I special create this method because for me and I think for rest of members of the team will be better for reading list_shoutouts_to(current_user) ~ (list shoutouts to current user) than read  list_shoutouts or user_shoutouts (list shoutouts/user shoutouts). For me this is not understand

Reply:
`current_user.shoutouts` is preferred to `Shoutout.where(user_id: current_user.id)`, and there can also be a way created to get a user's received shoutouts in a similar fashion (like `current_user.received_shoutouts`)